cmd/root.go

```diff
@@ -161,6 +161,20 @@ func init() {
 		"Set a request timeout for the client to connect with Kong (in seconds).")
 	viper.BindPFlag("timeout",
 		rootCmd.PersistentFlags().Lookup("timeout"))
+
+	rootCmd.PersistentFlags().String("tls-client-cert", "",
+		"Set the Kong's Admin TLS client certificate.\n"+
+			"This value can also be set using DECK_TLS_CLIENT_CERT "+
+			"environment variable. Must be used in conjunction with tls-client-key")
+	viper.BindPFlag("tls-client-cert",
+		rootCmd.PersistentFlags().Lookup("tls-client-cert"))
+
+	rootCmd.PersistentFlags().String("tls-client-key", "",
+		"Set the Kong's Admin TLS client key.\n"+
+			"This value can also be set using DECK_TLS_CLIENT_KEY "+
+			"environment variable. Must be used in conjunction with tls-client-cert")
```

Comment:
Since these 2 flags are expected to be used in conjunction, can you please add a check to verify that either both or none are passed, and error out if only one of them is used?

Reply:
Sure, will do.

Reply:
Done.